contracts/v1/event.sol

```diff
@@ -7,7 +7,11 @@ interface ListInterface {
 
 contract InstaEvent {
 
-    address public constant instaList = 0x0000000000000000000000000000000000000000;
+    address public immutable instaList;
+
+    constructor (address _instaList) public {
+        instaList = _instaList;
+    }
```

Comment:
We anyway don't need this contract. Right? We're not deploying v1 DSAs and v2 DSAs will be v1 on Polygon

Reply:
Just to make things easy for on sdk. And will have a common structure on it.